deployment/troposphere/app_template.py

```diff
@@ -26,13 +37,13 @@
     Description='Application server AMI'
 ))
 
-app_server_instance_profile_param = t.add_parameter(Parameter(
-    'AppServerInstanceProfile', Type='String',
-    Default=
-    'arn:aws:iam::900325299081:instance-profile/AppServerInstanceProfile',
-    Description='Physical resource ID of an AWS::IAM::Role for the '
-                'application servers'
-))
+# app_server_instance_profile_param = t.add_parameter(Parameter(
+#     'AppServerInstanceProfile', Type='String',
+#     Default=
+#     'arn:aws:iam::900325299081:instance-profile/AppServerInstanceProfile',
+#     Description='Physical resource ID of an AWS::IAM::Role for the '
+#                 'application servers'
+# ))
```

Comment:
This is pending until NYC can create IAM roles for us to assign to the launch configurations. We don't have write permission on IAM in the AWS accounts they gave us.

See also: https://github.com/azavea/nyc-trees/issues/242